tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1210,13 +1197,13 @@ public function userHasSharedSpace(
 		}
 		$fullUrl = $baseUrl . "ocs/v2.php/apps/files_sharing/api/v1/shares";
 
-		return HttpRequestHelper::post($fullUrl, "", $user, $this->featureContext->getPasswordForUser($user), [], $body);
+		HttpRequestHelper::post($fullUrl, "", $user, $this->featureContext->getPasswordForUser($user), [], $body);
 
 		$this->featureContext->theHTTPStatusCodeShouldBe(
 			200,
 			"Expected response status code should be 200"
 		);
-		$this->OCSContext->theOCSStatusCodeShouldBe(400, "Expected OCS response status code should be 200");
+		$this->featureContext->theOCSStatusCodeShouldBe(400, "Expected OCS response status code should be 200");
```

Comment:
This was an existing bug that was not noticed, because this code was previously not being executed.